data/js/tabs.js

```diff
@@ -4,13 +4,23 @@
   var config = (document.currentScript || {}).dataset || {}
   var forEach = Array.prototype.forEach
 
+  document.addEventListener('asciidoctor-tabs:toggle', function (event) {
+    const groupId = event.detail.syncGroupId.replace(/\W/g, '-').toLowerCase()
+    const id = event.detail.syncId.replace(/\W/g, '-').toLowerCase()
+    document.querySelectorAll('[class^="' + groupId + '"]').forEach(function (el) {
+      const hide = !el.classList.contains(groupId + '-' + id)
+      el.classList[(el.hidden = hide) ? 'add' : 'remove']('is-hidden')
+    })
+  })
+
   init(document.querySelectorAll('.tabs'))
 
   function init (tabsBlocks) {
     if (!tabsBlocks.length) return
     forEach.call(tabsBlocks, function (tabs) {
       var syncIds = tabs.classList.contains('is-sync') ? {} : undefined
       var tablist = tabs.querySelector('.tablist ul')
+      if (!tablist) return
```

Comment:
Just _The Boy Scout Rule_. Fixed an exception that is thrown when I had a `[.tags]` paragraph on my page.